pkg/services/ngalert/notifier/channels/victorops.go

```diff
@@ -94,7 +94,7 @@ func (vn *VictoropsNotifier) Notify(ctx context.Context, as ...*types.Alert) (bo
 	tmpl, _ := TmplText(ctx, vn.tmpl, as, vn.log, &tmplErr)
 
 	messageType := strings.ToUpper(tmpl(vn.MessageType))
-	if messageType == "" {
+	if messageType == "" || tmplErr != nil {
```

Comment:
I really dislike this kind of error handling and conditional assignment TBH, but since this is likely to be changed in the near future, it's ok.